diab_medications/migrations/0001_initial.py

```
@@ -18,19 +19,19 @@
-                ('route', models.CharField(max_length=20)),
+                ('frequency', models.CharField(choices=[('QD', 'Once daily'), ('BID', 'Twice daily'), ('TID', 'Three times daily'), ('QID', 'Four times daily'), ('Q6H', 'Every 6 hours'), ('Q8H', 'Every 8 hours'), ('Q12H', 'Every 12 hours'), ('PRN', 'As needed'), ('QW', 'Weekly'), ('QM', 'Monthly')], default='QD', max_length=50)),
                 ('start_date', models.DateField()),
                 ('end_date', models.DateField(blank=True, null=True)),
-                ('patient', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='patients_core.patient')),
-                ('encounter', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to='diab_encounters.encounter')),
+                ('encounter', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='medication_orders', to='diab_encounters.encounter')),
+                ('patient', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='medication_orders', to='patients_core.patient')),
             ],
             options={
                 'ordering': ['-start_date', '-id'],
+                'indexes': ["models.Index(fields=['patient', 'start_date'], name='diab_medica_patient_d7f8e7_idx')", "models.Index(fields=['patient', 'end_date'], name='diab_medica_patient_8c9f3e_idx')", "models.Index(fields=['atc'], name='diab_medica_atc_1a2b3c_idx')"],
```

Ruff (0.12.2)

Line too long (271 > 88)

(E501)